docs/framework/database/schema-design.md

```diff
@@ -13,10 +13,10 @@ Another method of viewing CiviCRM's Database model is to use one of various MySQ
 1. Look at the list of tables in the database. The table name generally gives you a good idea of the type of data it stores.
 2. Pick one table and look closer. Let's look at `civicrm_country` (a relatively simple example) by opening the "Table inspector" for that table.
     - We can look at the list of columns in the table.
-    ![MySQL Workbench CiviCRM Country Columns](/img/mysql_workbench_civicrm_country_tables.png)
+    ![MySQL Workbench CiviCRM Country Columns](/docs/img/mysql_workbench_civicrm_country_tables.png)
```

Comment:
I don't think that there is anything wrong on your end because this is the repo for your online documentation and it is not the supposed to be main source. As soon as I heard of this project, I immediately looked up the CiviCRM repos, found the dev docs, then I got excited and over-zealous whereas most of your visitors would probably find these topics the "proper" way:)

Thanks again!